modules/programs/zellij.nix

```diff
@@ -248,7 +281,16 @@ in
           "zellij/config.kdl" =
             mkIf (cfg.settings != { } && (lib.versionAtLeast cfg.package.version "0.32.0"))
               {
-                text = lib.hm.generators.toKDL { } cfg.settings;
+                text =
+                  (lib.hm.generators.toKDL { } cfg.settings)
+                  + lib.optionalString (cfg.extraConfig != "") (
+                    ''
+
+                      // extraConfig
+
+                    ''
```

Comment:
Not sure we really need a comment explaining which option the config came from.

Reply:
I guess we don't, it's just that in case there's conflicts between the generated config and the "extra" config, it'll be a bit more obvious that the bottom half of the config comes from the "extra" part.

I've found having this reminder helpful in cases where the generated config grows as the nix module grows and starts configuring something I had configured myself in the "extra" section before. This makes the weird conflict-related error that points me into the config file easier to debug.

I don't feel too strongly about having to keep it in the module. It's just something that in my mind may be useful and I'll do on my end anyway as I also like including the paths to the relevant files and docs.